Students/Ghiurutan V. Alexandru/Polynomial/src/Polynomial.java

```diff
@@ -0,0 +1,48 @@
+import java.text.DecimalFormat;
+
+public class Polynomial {
+	private double[] coeffs;
+
+	public Polynomial(double... coeffs) {
+		this.coeffs = coeffs;
+	}
+
+	public double[] getPolynomial() {
+		return coeffs;
```

Comment:
Another solution would be to have a separate method called `getDeepClonedPolynomial` which doesn't return the array of coefficients but a copy of it.